examples/features/csm_observability/README.md

```diff
@@ -0,0 +1,33 @@
+# CSM Observability
+
+This examples shows how to configure CSM Observability for gRPC client and
+server applications (configured once per binary), and shows what type of
+telemetry data it can produce for certain RPCs with additional CSM Labels. The
+gRPC Client accepts configuration from an xDS Control plane as the default
+address that it connects to is "xds:///helloworld:50051", but this can be
```

Comment:
What I don't understand is how is this example supposed to work without the user running a management server, given that the default dial target has an `xds` scheme.

Reply:
Added comment about needing to run in Cloud Service Mesh.